### PR TITLE
Added anchors sample

### PR DESCRIPTION
**Pull request by fordacious**
Added a simple anchors sample to demonstrate and validate API usage. Based off work by @bialpio

### CODE REVIEW

**Comment by bialpio** [comment]
Looks good! Do we also want to include anchor creation off of hit test results, or do you think that'd fit better in the hit test sample?

**Comment by lehoangphu** [comment]
There is a separate sample on hittest that combine hittest and anchor.  I will have the PR out soon.

**Review by bialpio — APPROVED** [review]
Approving in case you're waiting on me, but note that I do not have permissions to merge the PR.

**Comment by fordacious** [comment]
@toji

File: anchors.html [hunk 1]
```diff
@@ -0,0 +1,292 @@
+<!doctype html>
+<!--
+Copyright 2018 The Immersive Web Community Group
```
Comment (toji) [inline anchors.html:3]:
```suggestion
Copyright 2021 The Immersive Web Community Group
```

**Comment by toji** [comment]
Thanks! And thank you @bialpio for the review!

**Pull request merged**